03.bowling/bowling.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+FRAMES = 10
+
+def generate_pins
+  frames_text = ARGV[0]
+  frames = frames_text.split(',')
+  pins = []
```

Comment:
```suggestion
  pins_text = ARGV[0]
  pins = pins_text.split(',')
  frames = []
```

- `1,2,3,4...`の1や2はpin
- `[1,2]`や`[3,4]`のようにpinが1〜3個ごとにグルーピングされたものがframe
- `[[1,2],[3,4]]`のようにframeを要素として抱えている配列がframes

と名付けるのがいいんじゃないかなーと思います。
（それを踏まえてこのメソッドの名前も見直してみてください）

Reply:
入力で渡されるのがframeに分けられていないpinなので`pins_text`や`pins`に変更。
入力ご最終的にframeごとに分けた配列を渡すので格納する配列を`frames`に変更。